python/arcticdb/version_store/library.py

```diff
@@ -948,7 +948,7 @@ def sort_and_finalize_staged_data(
             new timeseries. Append collects the staged data and appends them to the latest version.
 
         prune_previous_versions : `Optional[bool]`, default=False
```

Comment:
The type hint is `Optional[bool]` on some methods and `bool` on others. I think it should be `bool` throughout.

Reply:
Made `bool` everywhere